html-web-page/index.html

```diff
@@ -25,8 +25,8 @@
     </style>
   </head>
   <body>
-    <h1>Jelly of the Month Club</h1>
-    <h2>Make all you friends jelly of your stock of jelly!</h2>
+    <header>Jelly of the Month Club</header>
+    <h1>Make all you friends jelly of your stock of jelly!</h1>
```

Comment:
Instead of putting the text directly inside the `header`, the whole `h1` should be inside the `header`